pwndbg/commands/heap.py

```diff
@@ -1641,17 +1650,15 @@ def jemalloc_heap() -> None:
 
     try:
         rtree = jemalloc.RTree.get_rtree()
-    except pwndbg.dbg_mod.Error as what:
-        # Fixes test_commands[jemalloc_heap] test case
-        print(message.warn(f"{what}"))
-        return
-
-    extents = rtree.extents
-    if len(extents) == 0:
-        print(message.warn("No extents found"))
+        extents = rtree.extents
+        if len(extents) == 0:
+            print(message.warn("No extents found"))
+            return
+        for extent in extents:
+            # TODO: refactor so not create copies
```

Comment:
wdym by copies here?

Reply:
fwiw this is me refactoring code to add try except only. Original code is by @jetchirag and I haven't reviewed that part closely yet, so not sure.